src/locale/locales/hu/messages.po

```diff
@@ -613,14 +613,14 @@ msgstr "A felnőtt tartalmak le vannak tiltva."
 #: src/view/com/composer/labels/LabelsBtn.tsx:139
 #: src/view/com/composer/labels/LabelsBtn.tsx:197
 msgid "Adult Content labels"
-msgstr "Felnőtt tartalmi feljegyzések"
+msgstr "Felnőtt tartalmú feljegyzések"
```

Comment:
I think there is a misunderstanding here. The sentence structure here is [Felnőtt] [tartalmi feljegyzések], not [Felnőtt tartalmi] [feljegyzések] – are you confusing ‘feljegyzés‘ (label) with ‘bejegyzés’ (post)?